src/reducers/user/user.spec.js

```diff
@@ -2,6 +2,10 @@ import { GET_USER_PROFILE } from '../../actionTypes';
 import userReducer from './';
 
 describe('User Reducer', () => {
+  it('returns default state when state parameter is not passed', () => {
+    expect(userReducer(undefined, {})).toEqual(null);
```

Comment:
Why is the default state null?

Reply:
@dinobi I have removed this implementation. It now resides in Aderokede's PR.